packages/fxa-content-server/app/scripts/lib/verification-methods.ts

```diff
@@ -8,9 +8,11 @@
  * In the future SMS and TOTP could also be included.
  */
 
-export default {
-  EMAIL: 'email',
-  EMAIL_2FA: 'email-2fa',
-  EMAIL_CAPTCHA: 'email-captcha',
-  TOTP_2FA: 'totp-2fa'
+enum VerificationMethods {
```

Comment:
The Programming Typescript book recommends using const enum's to opt into a stricter form of enums that doesn't allow key lookup that may be unsafe.

Reply:
Updated.

Reply:
Oh, we have to do key lookup, I reverted that change.